src/lib/holocene/icon/paths.ts

```diff
@@ -6,6 +6,8 @@ import arrowUp from './svg/arrow-up.svelte';
 import ascending from './svg/ascending.svelte';
 import astronaut from './svg/astronaut.svelte';
 import batchOperation from './svg/batch-operation.svelte';
+import beaker from './svg/beaker.svelte';
```

Comment:
We also have a very similar one to this under [labs](https://holocene-git-add-icons.preview.thundergun.io/?path=/story/icon--icon&args=name:labs).